app/components/supplementary-file-browser.js

```diff
@@ -42,14 +42,16 @@ export default Component.extend(Analytics, {
     hasNext: function() {
         return this.get('endIndex') < this.get('files.length');
     }.property('files', 'endIndex', 'startIndex'),
+    /* eslint-enable ember/no-function-prototype-extensions */
 
     __files: function() {
         this.set('files', []);
         this.set('selectedFile', null);
+        /* eslint-disable ember/named-functions-in-promises */
```

Comment:
this was rewritten once and caused errors. Decided to disable for the time being. Can try to fix again on second pass

Reply:
This one is super annoying to do :/. I prefer to refactor to use ember-concurrency because it's easier to read.
https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/named-functions-in-promises.md